custom/opm/reports.py

```diff
@@ -743,34 +737,13 @@ def get_block_filter(block):
         elif self.case_status == 'closed':
             query = query.filter(case_es.closed_range(lte=self.datespan.enddate_utc))
 
-        # TODO for consistency, we could always filter on awc using
-        # get_matching_awcs from the NewHealthStatusReport
-        if self.awcs:
-            query = query.filter(get_awc_filter(self.awcs))
-        elif self.gp:
-            query = query.filter(get_gp_filter(self.gp))
-        elif self.block:
-            query = query.filter(get_block_filter(self.block))
+        query = query.owner([user['doc_id'] for user in self.users_matching_filter])
 
         result = query.run()
 
-        def _final_filter(case_doc):
-            """
-            Because the ES filters are tokenized, we do a final in-memory filter ensuring exact matches.
-            This prevents issues with "Tetua Pasi Tola" cases showing up in "Tetua"
-
-            We can't do this in ES because we don't want to bother modifying the report_cases mapping.
-            See: http://www.elasticsearch.org/guide/en/elasticsearch/guide/current/_finding_exact_values.html
-            """
-            if self.awcs and case_doc.get('awc_name') not in self.awcs:
-                return False
-            elif self.block and case_doc.get('block_name') != self.block:
-                return False
-            return True
-
```

Comment:
This filter is no longer necessary, as we're now always selecting cases by owner_id, not `awc_name` or `block_name`